### PR TITLE
Ft confirmation mail formbuilder

### PR DESCRIPTION
**Pull request by megasnort**
The pull request for the confirmation mail in the FormBuilder, with better naming of the variables and fields:

- send_confirmation_mail
- confirmation_mail_subject
- confirmation_mail_content

### CODE REVIEW

**Comment by WouterSioen** [comment]
This can't be automatically merged, probably because you didn't follow the new bootstrap layout?

**Comment by megasnort** [comment]
Nope, I didn't I'll fix it.
I didn't know It would be implemented that soon (which is great)

File: src/Backend/Modules/FormBuilder/Js/FormBuilder.js [hunk 1]
```diff
@@ -58,6 +60,26 @@ jsBackend.formBuilder =
...
+ * Toggle opacity of confirmation e-mailfields.
+ * We do not use 'disabled' because it should be possible to alter the content of the textfields anyway
+ */
+jsBackend.formBuilder.confirmationEmail = {
+	init: function()
+	{
+		jsBackend.formBuilder.confirmationEmail.disOrEnableFields();
+		$('#sendConfirmationMail').change(jsBackend.formBuilder.confirmationEmail.disOrEnableFields);
+	},
+	disOrEnableFields: function()
```
Comment (turanct) [inline src/Backend/Modules/FormBuilder/Js/FormBuilder.js]: dis- or enable? :-D just make it `disableOrEnableFields`

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php [hunk 2]
```diff
@@ -523,21 +547,33 @@ private function validateForm()
...
+                // Confirmation e-mail to person who filled in the form
+                // Double check for empty subject or content and make sure the e-mail is allowed to be send.
+                // Also, make sure we have a valid reply to e-mail adres (which is checked above)
+                if (
+                    trim($this->item['confirmation_mail_subject']) != '' &&
+                    strip_tags(trim($this->item['confirmation_mail_content'])) != '' &&
+                    $this->item['send_confirmation_mail'] == 'Y' &&
+                    $replyToGiven
+                ) {
+                    $message = \Common\Mailer\Message::newInstance($this->item['confirmation_mail_subject'])
```
Comment (turanct) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: The fully qualified classname `\Common\Mailer\Message` is used a lot in this file, would be better if there was a `use` statement

**Comment by megasnort** [comment]
Should be fine now.

**Comment by megasnort** [comment]
Sorry, somehow missed that.

Should be fine now. :)

**Comment by WouterSioen** [comment]
We'll merge this later, but we'll merge bootstrap first.

**Pull request closed without merging**